docs/user-guide/analysis.md

```diff
@@ -59,15 +63,167 @@ be found
 
 #### Example: Reserving a single node for 20 minutes for interactive analysis
 
+    ```bash
     auser@uan01:> salloc --nodes=1 --tasks-per-node=1 --cpus-per-task=1 \
                   --time=00:20:00 --partition=standard --qos=short \
                   --reservation=shortqos --account=[budget code]
-
+    ```
 
 ## Data analysis nodes
 
-The full ARCHER2 system will have dedicated data analysis nodes. Further
-information on these nodes will be added here once it becomes available.
+The data analysis nodes on the ARCHER2 full system are designed for large
+compilations, post-calculation analysis and data manipulation. They should 
+be used for jobs which are too small to require a whole compute node, but 
+which would have an adverse impact on the operation of the login nodes if 
+they were run interactively.
+
+Unlike compute nodes, the data analysis nodes are able to access the `/home`, 
+`/work`, and `/rdfaas` directories. They can also be used to transfer data 
```

Comment:
I think it is RDFaaS /epsrc and /general file systems rather than /rdfaas directories